orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -155,17 +155,17 @@ def test_forward_call_multiple_positional_arguments():
     output = model(x, y)
     assert output is not None
 
-def test_forward_call_positional_arguments():
-    device = 'cuda'
+# def test_forward_call_positional_arguments():
```

Comment:
Is there a new regression in this test? (Looking at the implementation, I can't see an obvious reason why it would interact with this PR.)

Reply:
no... the UT is currently broken in the dev branch. will revert this change once it's fixed in dev branch .